src/main/java/com/rakovets/course/java/core/practice/oop/principles/hero/Warrior.java

```diff
@@ -0,0 +1,20 @@
+package com.rakovets.course.java.core.practice.oop.principles.hero;
+
+public class Warrior extends Hero{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"